src/main/kotlin/org/rust/lang/core/completion/RsAttributeCompletionProvider.kt

```diff
@@ -70,11 +65,12 @@ object RsAttributeCompletionProvider : RsCompletionProvider() {
         onStructLike to "non_exhaustive"
     ).flatMap { entry -> entry.value.split(' ').map { attrName -> RustAttribute(attrName, entry.key) } }
 
-    override fun addCompletions(parameters: CompletionParameters,
-                                context: ProcessingContext,
-                                result: CompletionResultSet) {
-
-        val elem = parameters.position.superParent(RsPsiPattern.META_ITEM_IDENTIFIER_DEPTH)
+    override fun addCompletions(
+        parameters: CompletionParameters,
+        context: ProcessingContext,
+        result: CompletionResultSet
+    ) {
+        val elem = context[META_ITEM_ATTR]?.parent ?: return
```

Comment:
Btw, this is incorrect in the case of an inner attribute inside a function body. We probably should use `RsAttr.owner`, but it's also buggy in the case of inline modules (for example) =/

Reply:
Fixed `RsAttr.owner` in #6453

Reply:
👍